packages/plugin-js-packages/src/lib/runner/index.ts

```diff
@@ -69,16 +72,29 @@ async function processOutdated(
     ignoreExitCode: true, // outdated returns exit code 1 when outdated dependencies are found
   });
 
+  const totalDeps = objectFromEntries(
+    await Promise.all(
+      depGroups.map(async depGroup => [
+        depGroup,
+        await getTotalDependencies(depGroup, packageJsonPath),
+      ]),
+    ),
```

Comment:
:snail: This would reads the same `package.json` file multiple times. Reading from disk is in general not a cheap operation, so it would be more optimal to read the file once, since it contains all the dependency groups.

Reply:
Good point, improved.